settings.gradle.kts

```
@@ -178,22 +178,14 @@ FeaturePreviews.Feature.values().forEach { feature ->
 
 fun remoteBuildCacheEnabled(settings: Settings) = settings.buildCache.remote?.isEnabled == true
 
-fun isAdoptOrAdoptiumOpenJDK() = true == System.getProperty("java.vendor")?.let { it.contains("AdoptOpenJDK") || it.contains("Adoptium") }
-
-fun isAdoptOrAdoptiumOpenJDK11() = isAdoptOrAdoptiumOpenJDK() && JavaVersion.current().isJava11
-
 fun getBuildJavaHome() = System.getProperty("java.home")
 
 gradle.settingsEvaluated {
     if ("true" == System.getProperty("org.gradle.ignoreBuildJavaVersionCheck")) {
         return@settingsEvaluated
     }
 
-    if (!isAdoptOrAdoptiumOpenJDK()) {
-        if (remoteBuildCacheEnabled(this)) {
-            throw GradleException("Remote cache is enabled, which requires AdoptOpenJDK 11 to perform this build. It's currently ${getBuildJavaHome()}.")
-        } else {
-            println("WARNING: you're running this build on ${getBuildJavaHome()}, which is not officially supported")
-        }
+    if (!JavaVersion.current().isJava11) {
+        throw GradleException("This build requires JDK 11. It's currently ${getBuildJavaHome()}.")
```

```suggestion
        throw GradleException("This build requires JDK 11. It's currently ${getBuildJavaHome()}. You can ignore this check by passing '-Dorg.gradle.ignoreBuildJavaVersionCheck'.")
```

Let's help folks who want to run the build with a different Java version.